app/feedme/versioning_service.py

```
@@ -35,393 +34,151 @@
-            if edit_request.metadata is not None:
+            if edit_request.metadata:
                 update_data['metadata'] = edit_request.metadata
             
-            # Create new version
-            new_version = self.create_new_version(
+            await self.supabase_client.update_conversation(
                 conversation_id=conversation_id,
-                updated_data=update_data,
-                updated_by=edit_request.updated_by
+                update_data=update_data
             )
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'update_data' in method call

(E1123)